packages/migrations/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "4.3.2",
+        "changes": [
+            {
+                "note": "Removed dependency on @0x/order-utils",
```

Comment:
I had to introduce a dependency on `@0x/migrations` to `@0x/order-utils` for tests (which were retrieved from the old exchange wrapper tests). I've run into similar issues with tests introducing circular dependencies for other packages. Would like to discuss the possibility of developing an integration test suite for any tests that run with ganache and the standard 0x migration. This would also potentially speed up our testing time, by doing one migration + snapshots instead of separate migrations in numerous unit test suites. 

Since we intend to remove these methods in a couple months, this seems like the easiest solution for now to unblock this task.